modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -466,7 +466,8 @@ public static void removeBinding(Binding binding) throws AndesException {
      * @param subscription subscription
      * @throws AndesException
      */
-    private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue, Subscription subscription) throws AndesException, SubscriptionAlreadyExistsException {
+    private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue,
+                                                                   Subscription subscription) throws SubscriptionAlreadyExistsException, AMQException, AndesException {
```

Comment:
If you want to pass it you can create an AMQException from the AndesException and pass?